src/DotNetLightning.Core/Serialization/Msgs/Msgs.fs

```diff
@@ -1809,6 +1818,50 @@ and WhichChannel =
     | SpecificChannel of ChannelId
     | All
 
+
+[<RequireQualifiedAccess>]
+[<CLIMutable>]
+type WarningMsg =
+    {
+        mutable ChannelId: WhichChannel
+        mutable Data: array<byte>
+    }
+
+    interface ISetupMsg
+
+    interface ILightningSerializable<WarningMsg> with
+        member this.Deserialize ls =
```

Comment:
@webwarrior-ws let's use a better name than "ls" here

Reply:
again, seems to be the convention in the codebase